src/clouds.cpp

```diff
@@ -191,7 +193,10 @@ void Clouds::render()
 					(float)p_in_noise_i.X * cloud_size_noise,
 					(float)p_in_noise_i.Y * cloud_size_noise,
 					m_seed, 3, 0.5);
-			grid[i] = (noise >= 0.4);
+			// normalize to 0..1 (given 3 octaves)
+			double noise_bound = 1.0 + 0.5 + 0.25;
+			double density = noise / noise_bound * 0.5 + 0.5;
+			grid[i] = (density < m_density);
```

Comment:
Ok, "above 0.4" works out to "above 61%", which is quite close to "below 40%", assuming a symmetric _and uniform_ distribution. It's not uniform, though, so it might be a tad different.

Use the mini-mod I posted and try densities that look right ;-)